### ISSUE TITLE
Bug in SVG implementation

### ISSUE THREAD

**Issue by gudmdharalds** [Automattic/VIP-Coding-Standards#218]
When scanning a SVG file saved in, say, `/tmp/testingphpimplementation.svg`, and the file contains issues that should be caught, the current implementation will not turn up anything.

To replicate:

* Put the following in the file `/tmp/testingphpimplementation.svg`:

```
<svg>
<?php

echo esc_html("test");

?>
</svg>
```

And then scan:

```
phpcs   --standard='WordPress-VIP-Go' --severity='1' --report='json' --extensions='svg' --sniffs='WordPressVIPMinimum.SVG.HTMLCode' /tmp/testingsomephpstuff.svg
```

No issues are reported.

Solution:

* Remove this line here: https://github.com/Automattic/VIP-Coding-Standards/blob/2f22ed1849077473aa925735b6e88a25958f4374/WordPressVIPMinimum/ruleset.xml#L22, or
*  Escape the `.`

**Comment by gudmdharalds** [comment]
Issue #178 is related to this.

**Comment by tomjn** [comment]
Would this actually run on the server?

**Comment by gudmdharalds** [comment]
Closing this, as support for SVG checking will be moved in `vip-go-ci`, see #220.
